site_root/input/pages/StructureDefinition-ncpi-phenotype-notes.md

```diff
@@ -0,0 +1,20 @@
+#### Observation.code
+The code property is described as "the type of observation" by HL7, and as such, we have stipulated that the code must originate from the [Human Phenotype Ontology](http://www.human-phenotype-ontology.org). For cases where there is no ideal HPO code to suite a researcher's needs, users can use a more general HPO code as the first entry in the code array and one or more codes from other systems which are more technically accurate. 
```

Comment:
Catching another minor spelling issue, "HPO code to suite a researcher's needs', should be suit not suite.

Reply:
Fixed!